pytorch_toolkit/tools/instantiate_template.py

```diff
@@ -19,6 +19,10 @@
 
 import yaml
 
+import sys
+# TODO: consider with Ilya: this trick is required when we instantiate template in a clean system without ote installed
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'ote'))
```

Comment:
2 @Ilya-Krylov :   
Ilya, please, pay attention to these lines.
I made several experiments with clean environment and found that this trick is really required.

Reply:
My quastion: is this trick ok?

Reply:
ok

Reply:
remove comment